docs/topics/gradle/gradle-configure-project.md

```diff
@@ -1150,6 +1150,39 @@ dependencies {
 </tab>
 </tabs>
 
+## Declare repositories
+
+To declare a publicly-available repository so that you can use its open source dependencies, set the name of the repository
+in the `repositories` block:
```

Comment:
Should we also mention [this centralized way](https://docs.gradle.org/current/userguide/declaring_repositories.html#sub:centralized-repository-declaration) to declare repositories?